### DISCUSSION TITLE
Syntax for command line use of flags

### DISCUSSION THREAD

**Discussion by tbarmann** [pdfcpu/pdfcpu#863] [Q&A] [answered]
I can't seem to find the correct syntax for using the `conf` flag when running pdfcpu from the command line when filling an existing pdf. https://pdfcpu.io/getting_started/common_flags.html 
I have tried:

```
pdfcpu form fill my.pdf out.json out.pdf -conf disable
pdfcpu -conf disable form fill in.pdf in.json out.pdf 
```

And many other variations. I keep getting the usage message:
`usage: pdfcpu form fill inFile inFileJSON [outFile]`

**Comment by hhrutter** [comment, marked as answer]
Put your options between the command and the positional parameters:

`pdfcpu form fill -conf disable my.pdf out.json out.pdf`

Even better using double dashes `--` to separate your options from the positional parameters.
Especially if you have `-` in your positional parameters:

`pdfcpu form fill -conf disable -- my.pdf out.json out-new.pdf`

**Reply by tbarmann** [reply]
Thank you!
